### PR TITLE
Change PowerReduction from 1 million to 1 hundred for testnet

### LINKED ISSUE #65: Change PowerReduction for testnet
Currently it is set to `1000000` which causes `1000000tree` tokens to actually have a voting power of `1`. We want to change this temporarily for `regen-test-1001` and then set it to a higher value and simply issue more tokens for the next testnet and mainnet.

### PR DESCRIPTION
**Pull request by aaronc**
Close #65

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -10,6 +10,11 @@ build:
   image: golang:1.11.5
   script:
     - make install
+  artifacts:
```
Comment (ethanfrey) [inline .gitlab-ci.yml]: Maybe this can go in another PR?

Doesn't seem directly related to the PR title

File: app/app.go [hunk 2]
```diff
@@ -68,6 +69,8 @@ func init() {
 		geo.AppModuleBasic{},
 		upgrade.AppModuleBasic{},
 	)
+
+	sdk.PowerReduction = sdk.NewIntFromBigInt(new(big.Int).Exp(big.NewInt(10), big.NewInt(2), nil))
```
Comment (ethanfrey) [inline app/app.go]: So... I dug quite a bit to the usage of this variable.

It seems that:

1. Setting it in the constructor is the best way to override AND
2. You do not actually need to re-calculate all the validators powers, as this is done every EndBlock automatically, so will be updated at the end of the first block after the upgrade.

```
// Apply and return accumulated updates to the bonded validator set. Also,
// * Updates the active valset as keyed by LastValidatorPowerKey.
// * Updates the total power as keyed by LastTotalPowerKey.
// * Updates validator status' according to updated powers.
// * Updates the fee pool bonded vs not-bonded tokens.
// * Updates relevant indices.
// It gets called once after genesis, another time maybe after genesis transactions,
// then once at every EndBlock.
//
// CONTRACT: Only validators with non-zero power or zero-power that were bonded
// at the previous block height or were removed from the validator set entirely
// are returned to Tendermint.
func (k Keeper) ApplyAndReturnValidatorSetUpdates(ctx sdk.Context) (updates []abci.ValidatorUpdate) {
```

File: app/app.go [hunk 3]
```diff
@@ -198,6 +201,7 @@ func NewXrnApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest bo
 	app.crisisKeeper = crisis.NewKeeper(crisisSubspace, invCheckPeriod, app.distrKeeper,
 		app.bankKeeper, app.feeCollectionKeeper)
 	app.upgradeKeeper = upgrade.NewKeeper(app.upgradeStoreKey, app.cdc)
+	app.upgradeKeeper.SetUpgradeHandler("regen-test-1000-upgrade-1", func(ctx sdk.Context, plan upgrade.Plan) { })
```
Comment (ethanfrey) [inline app/app.go]: I would add a comment here, saying this is a checkpoint for code upgrade - in particular PR #66 (powerreduction change)

File: app/app.go (hunk 2 above)
Comment (ethanfrey) [reply app/app.go]: Adding a short comment before this line would not hurt however.

File: .gitlab-ci.yml (hunk 1 above)
Comment (aaronc) [reply .gitlab-ci.yml]: Normally yes, but I want to have upgrade binaries automatically generated in CI. I tested it and it works...

Comment (aaronc) [reply .gitlab-ci.yml]: Okay if I keep it here for now @ethanfrey ?

File: app/app.go (hunk 2 above)
Comment (aaronc) [reply app/app.go]: Okay, so this seems like a correct way to do this? It's in an `init` function.

File: .gitlab-ci.yml (hunk 1 above)
Comment (ethanfrey) [reply .gitlab-ci.yml]: Sure. More a comment for style, and general PR flow.

But I see the urgency here and it's a super minor point. Let's get this out and merged.

File: app/app.go (hunk 2 above)
Comment (ethanfrey) [reply app/app.go]: Anything before the first `EndBlock` call should work well.

`init` is good to ensure that the default sdk value is never read anywhere
